code/mob/living/intangible/flock/flocktrace.dm

```diff
@@ -93,6 +93,11 @@
 
 /mob/living/intangible/flock/trace/death(gibbed, suicide = FALSE)
 	. = ..()
+	if (istype(src.loc, /mob/living/critter/flock/drone))
+		var/mob/living/critter/flock/drone/F = src.loc
+		F.release_control_abrupt(FALSE)
+		if (F.z == Z_LEVEL_STATION)
+			flock_speak(null, "Control of drone [F.real_name] surrended.", src.flock)
```

Comment:
why is the speaker null

Reply:
It's for a "general" system message, consistency with other messages we have